pkg/util/mount/mount_windows.go

```
@@ -0,0 +1,132 @@
+// +build windows
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mount
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+type Mounter struct {
+	mounterPath string
+}
+
+func (mounter *Mounter) Mount(source string, target string, fstype string, options []string) error {
+	if !strings.HasPrefix(target, "c:") && !strings.HasPrefix(target, "C:") {
+		target = "c:" + target
+	}
+
+	if source == "tmpfs" {
+		glog.Infof("azureMount: mounting source (%q), target (%q)\n, with options (%q)", source, target, options)
+		os.MkdirAll(target, 0755)
+		return nil
+	}
+
+	parentDir := filepath.Dir(target)
+	err := os.MkdirAll(parentDir, 0755)
+	if err != nil {
+		return fmt.Errorf("mkdir(%q) failed: %v", parentDir, err)
+	}
+
+	if len(options) != 1 {
+		glog.Warningf("azureMount: mount options(%q) command(%n) not equal to 1, skip mounting.", options, len(options))
+		return nil
```

Shouldn't we return an error here?